pytorch_lightning/trainer/connectors/model_connector.py

```diff
@@ -35,5 +35,4 @@ def copy_trainer_model_properties(self, model):
             m._device_type = str(self.trainer._device_type)
             m._distrib_type = str(self.trainer._distrib_type)
             m.use_amp = self.trainer.amp_backend is not None
-            m.testing = self.trainer.testing
```

Comment:
we need to provide a getter for that, right?

Reply:
What do you mean? For the model?

I think I only found a single place where `model.testing` was used. I don't think it's necessary

Reply:
I was thinking about BC for the users who use that in their model hooks.
Personally I wouldn't mind it be gone because conceptually it anyway conflicts with the `model.training` property that comes from `nn.Module`

Reply:
I think it's an artefact of the past

Like, that property is not updated every time testing changes
And why would we have testing but not all the others...
+ the conflict you mentioned with that from `nn.Module`

So let's not have it